Sources/YouboraConfig.swift

```diff
@@ -40,7 +40,7 @@ struct YouboraConfig: Decodable {
             options.contentIsLiveNoSeek = media.isDVR != nil ? NSNumber(booleanLiteral: !(media.isDVR!)) : nil
             options.contentDuration = media.duration != nil ? NSNumber(value: media.duration!) : nil
             options.contentTitle = media.title
-            options.contentTitle2 = media.title2
+            options.program = media.program != nil ? media.program : media.title2
```

Comment:
I just remove the ternary operator and the computed property as well because. Our lib already handle the contentTitle2 so I leave it has it was. In addition I check if media.program is diff from nil and case yes I defined the options.program with this media value. Otherwise without that condition the lib would auto define program when contentTitle2 was defined but the media.program would override it even if it was nil.